### PR TITLE
Add missing avatarSize property for the Page component

### PR DESCRIPTION
**Pull request by SmolinPavel**
This is the fix for the issue I've raised https://github.com/reactjs/reactjs.org/issues/1453

### CODE REVIEW

**Comment by lex111** [comment]
@alexkrolick please review this.

File: content/docs/context.md [hunk 1]
```diff
@@ -57,18 +57,17 @@ It might feel redundant to pass down the `user` and `avatarSize` props through m
 One way to solve this issue **without context** is to [pass down the `Avatar` component itself](/docs/composition-vs-inheritance.html#containment) so that the intermediate components don't need to know about the `user` prop:
 
 ```js
-function Page(props) {
-  const user = props.user;
+function Page({ user, avatarSize }) {
```
Comment (alexkrolick) [inline content/docs/context.md]: None of the other function components on the page use destructuring for props, can we keep this the same and just add the missing prop below?

Comment (SmolinPavel) [reply content/docs/context.md]: @alexkrolick Thanks for you reply. Sure, I'll update this PR to match the standards.

Comment (SmolinPavel) [reply content/docs/context.md]: @alexkrolick seems that it was fixed 23 hour ago 👍 https://github.com/reactjs/reactjs.org/commit/8e805d799501b2cc7db3a9726272f90228fc1916

Comment (SmolinPavel) [reply content/docs/context.md]: Closing this PR

**Pull request closed without merging**